analytics/build/build.go

```diff
@@ -43,6 +44,26 @@ func New(analytics *config.Analytics) analytics.Runner {
 			glog.Errorf("Could not initialize PubstackModule: %v", err)
 		}
 	}
+	glog.Info("pubxai.Enabled: ", analytics.Pubxai.Enabled)
```

Comment:
Is this log line necessary? Do the other analytics adapters do this?

Reply:
removed it